elements/order-input.html

```diff
@@ -3,131 +3,3 @@
 <link rel="import" href="../bower_components/paper-checkbox/paper-checkbox.html">
 <link rel="import" href="../bower_components/paper-input/paper-input.html">
 <link rel="import" href="../bower_components/iron-location/iron-location.html">
```

Comment:
what's the purpose of `order-input.html` now? If this file should be deleted, shouldn't we make it a git-move from `order-input.html` to `order-input.js`, rather than delete and add?